sigsci_site_manager/merge.py

```diff
@@ -296,31 +296,31 @@ def merge_integrations(api, data):
                 print('    Failed: %s' % e)
 
 
-def generate_advanced_rules_request(api, source, data):
+def merge_advanced_rules(api, source, data):
     print('Merging advanced rules...')
 
     # Get the existing advanced rules
     src = api.get_advanced_rules()
     rules = filter_data(src.get('data', []), ['shortName'])
     rule_names = [r['shortName'] for r in rules]
-
-    email = '\nEmail support@signalsciences.com with the following...\n'
-    email += ('Please copy the following advanced rules from %s/%s to %s/%s:' %
-              (source['corp'], source['site'], api.corp, api.site))
-    make_request = False
-    skipped = []
     for item in data:
         if item['shortName'] not in rule_names:
-            email += '\n    %s (ID %s)' % (item['shortName'], item['id'])
-            make_request = True
+            if '+' in item['shortName']:
+                print("    Warning for %s: Rule names containing '+' not supported."
+                      % item['shortName'])
```

Comment:
Same comments as above in the deploy code

Reply:
Changed it to just catch any failed copy and output an email request at the end.